appstoreconnect/api.py

```diff
@@ -70,25 +74,26 @@ def __len__(self):
 			def __next__(self):
 				if not self.payload:
 					self.fetch_page()
-
-				data = self.payload.get('data', [])[self.index]
-				self.index += 1
-				if self.index == len(self.payload.get('data', [])):
+				if self.index < len(self.payload.get('data', [])):
```

Comment:
I don't understand the changes here.
It seems to do the same thing as before but I find the code less readable.
Can you explain the idea behind these changes in the __next__ method ?

Reply:
After more testing I found out your code indeed fixed a bug with paging in the iterator (a resource was missing in the first page)
I think we can simplify code a bit but I'll keep your changes as-is for now.

Thanks again!